### PR TITLE
MTM-51374: document support for oauth2 access token for sso

### PR DESCRIPTION
**Pull request by chbakar**

### CODE REVIEW

**Comment by BeateRixen** [comment]
@chbakar Could we add a link to the documentation, that is, is this documented somewhere in the documentation?

**Comment by chbakar** [comment]
> @chbakar Could we add a link to the documentation, that is, is this documented somewhere in the documentation?

There is no mention of it in the user guide or any place else as customers don't really interface with it.

**Review by BeateRixen — APPROVED** [review]

File: content/release-10-17-0/whatsnew-10-17-0.md [hunk 1]
```diff
@@ -57,3 +57,10 @@ We have added a new option for dashboard configuration that allows users to rest
 ![Dashboard availability](/images/release-notes/dashboard-availability.png)
 
 For details, see [Cockpit > Working with dashboards](https://cumulocity.com/guides/10.17.0/users-guide/cockpit/#dashboards) in the *User guide*.
+
+
+### Support for new OAuth2 access token type in single sign-on authentication
+
+Cumulocity IoT supports new standard OAuth2 access tokens with the "at+jwt" type header parameter. For more information on this parameter, see [OAuth2 Access Tokens](https://datatracker.ietf.org/doc/html/rfc9068#name-header).
+Single-sign-on users managed by new authorization servers (that is, generating OAuth2 access tokens with type "at+jwt") can thus login successfully into Cumulocity IoT.
+To process this new token type, Cumulocity IoT uses the [Nimbus JOSE + JWT](https://connect2id.com/products/nimbus-jose-jwt).
```
Comment (sarangovindarajan) [inline content/release-10-17-0/whatsnew-10-17-0.md:66]: We can avoid the details about the new library introduced in this section.

**Review by sarangovindarajan — COMMENTED** [review]
We can avoid the details about the new library introduced in this section.

File: content/release-10-17-0/whatsnew-10-17-0.md (hunk 1 above)
Comment (chbakar) [reply content/release-10-17-0/whatsnew-10-17-0.md:66]: Then I'll remove the last sentence

**Review by sarangovindarajan — APPROVED** [review]

**Pull request merged**